qml/pages/VenueDescription.qml

```diff
@@ -217,7 +217,7 @@ BVApp.Page {
             font.pixelSize: BVApp.Theme.fontSizeSmall
             text: isFoodVenue ?
                       (Qt.locale().name.toLowerCase().indexOf("de") === 0 ? // startsWith() was introduced in Qt 5.8 and Sailfish is currently running 5.6
-                           restaurant.comment :
+                           (restaurant.review ? restaurant.comment + "<br><br>" + restaurant.review : restaurant.comment) : // if there is a German review, add if after the short comment
```

Comment:
Cool that you changed it in the way both the short comment and the review are visible if there is a review!
I just added a small suggestion (which I myself am not 100% sure whether this is the better solution code wise).

Independent of that I actually do think we should show the German review even if the device is switched to English (or Dutch). Reason: Just suppose someone is a native speaker and has their device naturally switched to English, but someone that does speak (a bit of) German. They would not even notice that our app has full length reviews and even if they know, they'd probably still want to view those reviews without switching their device to German.

(In the long run, I assume we need to provide a setting for those things).

Reply:
Good suggestions! I squashed it into "Use newly introduced 'review' field". Please re-review.